File: Hanyang-CSE-Assignments-main/2022_ite1014/2022_ite1014_2022048022-master/6-2/2/2.py
import random

def findMax(ls):
    largest = 0
    for i in range(100):
        if(ls[i]>largest):
            largest = ls[i]
    return largest

def findMin(ls):
    smallest = 987654321
    for i in range(100):
        if(ls[i]<smallest):
            smallest = ls[i]
    return smallest

my_list = []
for i in range(100):
    num = random.randint(1,1000)
    print(num, end = ' ')
    my_list.append(num)
print()

print("max value:", findMax(my_list))
print("min value:", findMin(my_list))

File: Hanyang-CSE-Assignments-main/2022_ite1014/2022_ite1014_2022048022-master/8-1/2/2.c
#include <stdio.h>

int main(){
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    if(a>=b && b>=c){
        printf("min: %d\n", c);
        printf("max: %d", a);
    }
    else if(a>=c && c>=b){
        printf("min: %d\n", b);
        printf("max: %d", a);
    }
    else if(b>=a && a>=c){
        printf("min: %d\n", c);
        printf("max: %d", b);
    }
    else if(b>=c && c>=a){
        printf("min: %d\n", a);
        printf("max: %d", b);
    }
    else if(c>=a && a>=b){
        printf("min: %d\n", b);
        printf("max: %d", c);
    }
    else if(c>=b && b>=a){
        printf("min: %d\n", a);
        printf("max: %d", c);
    }
}